sql/dbschema.sql

```diff
@@ -0,0 +1,26 @@
+CREATE TABLE users (
+  userId INT PRIMARY KEY IDENTITY(1,1),
+  firstName VARCHAR(50) NOT NULL,
+  lastName VARCHAR(50) NOT NULL,
+  email VARCHAR(50) NOT NULL,
+  deleted INT DEFAULT 0 NOT NULL
+);
+
+CREATE TABLE tasks(
+  taskId INT PRIMARY KEY IDENTITY(1,1),
+  userId INT NOT NULL,
+  title VARCHAR(50),
+  description VARCHAR(400),
+  createdTime DATETIME NOT NULL,
+  deleted INT NOT NULL DEFAULT 0,
+  completed INT NOT NULL DEFAULT 0
+
+  CONSTRAINT FK_User_Task FOREIGN KEY (userId)
+    REFERENCES  users(userId)
+    ON DELETE CASCADE
```

Comment:
This probably makes sense to include in the tasks table setup, but if we never actually delete users would this ever come into play?

Reply:
no, but if we decide to delete a user in the future, we won't be worried about modifying the tasks table. do you think it's not necessary here?

Reply:
I'd say the simpler the better this time around, so that we do not run into 500 internal server errors when we delete stuff and cannot.